client/components/domains/domain-search-results/index.jsx

```diff
@@ -245,7 +245,7 @@ class DomainSearchResults extends React.Component {
 
 	render() {
 		return (
-			<div className="domain-search-results">
+			<div className="domain-search-results" aria-live="polite" aria-relevant="additions">
```

Comment:
Technically this works, but there's a slight delay between the paid domains & the "free" option, which means that the screen reader is in the process of reading the first paid result when the free result appears, and so it ends up interrupting itself to read the free row, then stops.

Another solution to this might be something like this, which would allow for live feedback without having to read the entire table to the screen reader.

```
<div aria-live="polite">
    <ScreenReaderText>5 domains found</ScreenReaderText>
</div>
```

Reply:
👍

Reply:
and any way to create a link between that counter and the actual search results?

Reply:
I'm not sure what you mean– the counter would basically be an audio "notice", the user likely wouldn't intentionally navigate focus to it. once they've typed out their search, they would navigate into the search results.

Reply:
> and any way to create a link between that counter and the actual search results

If you mean how to get the count, the component should have access to that. I suppose it'd be something like domain availability + domain suggestions (when they've loaded)

Reply:
I've updated the code on 7a4fe27f76fba7a3ea7060f0e7e69073ee093320 with `ScreenReaderText`. Now there are two `aria-live` regions:

- Domain Availability / Result or TLD not available.
- Result count.

I think I've covered all the cases (with or without suggested domains, weird TLDs, owned domains, etc), and I think that the voiceover makes sense.
The only problem I think we have is that we always show 10 domain suggestions for the signup flows, and for the domain add flow on _My Sites_ we initially show 2, and then 10.
The suggestions could have a WordPress.com domain or not (like on _My Sites / Domains_), but it's always 10.
Does this matter? Should we include a more descriptive text?

Reply:
Is it super important that the count is correct? Could we maybe phrase text to say, something along the lines of domain results found, tab to examine/review results?

Reply:
No, the count doesn't really matter, the point is to have some kind of audio feedback - so that would work.

Reply:
Apologies, @benhuberman we'll try and remember to add a direct comment for y'all when adding the `needs copy review` label. In this case I think we need some help with text that lets folks know that a search result has completed, without needing to describe an exact count. Eg 

```jsx
<div aria-live="polite">
    <ScreenReaderText>Domains found! Tab to select an option. </ScreenReaderText>
</div>
```

Reply:
Thanks @gwwar! It's sometimes hard to propose copy without more context about the flow or a screenshot that shows how it's displayed. In this case, though, it seems like a fairly minor / straightforward case... how about:
> Here are some domain ideas -- tap to select one of the options.

Let me know if this doesn't quite fit (space-wise or otherwise)!

Reply:
@benhuberman I'm not sure that works for this case, here's some more context: This text won't be shown visually- it'll be read aloud to screen reader users when the domain results change, so that when they're typing in the search box, there is audio feedback that the results have updated. Then they can use the keyboard to get to the results.